File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/Sykefraværsstatistikk.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller;

import java.math.BigDecimal;

public interface Sykefraværsstatistikk {

  int getÅrstall();

  int getKvartal();

  int getAntallPersoner();

  BigDecimal getTapteDagsverk();

  BigDecimal getMuligeDagsverk();
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/Sykefraværsstatistikk.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

import java.math.BigDecimal

interface Sykefraværsstatistikk {
    @JvmField
    val Årstall: Int
    @JvmField
    val kvartal: Int
    @JvmField
    val antallPersoner: Int
    @JvmField
    val tapteDagsverk: BigDecimal?
    @JvmField
    val muligeDagsverk: BigDecimal?
}
